### ISSUE TITLE
Update Spyder but Spyder Does Not Update

### ISSUE THREAD

**Issue by HJLBX** [spyder-ide/spyder#8980]
### What steps will reproduce the problem?

Admin Conda console >

conda update qt pyqt
conda update spyder

Admin console states everything already installed, but Spyder actually does not update - still remains at 3.3.2 and every time open it states 3.3.3 available:

https://imgur.com/a/Le5NeUz

## Versions

* Spyder version: 3.3.2 
* Python version: 3.7.1
* Qt version: 5.9.6
* PyQt5 version: 5.9.2
* Operating System: Windows 10

### Dependencies

```
pyflakes >=0.6.0  :  2.0.0 (OK)
pycodestyle >=2.3 :  2.4.0 (OK)
pygments >=2.0    :  2.3.1 (OK)
sphinx >=0.6.6    :  1.8.2 (OK)
rope >=0.9.4      :  0.11.0 (OK)
jedi >=0.9.0      :  0.13.2 (OK)
nbconvert >=4.0   :  5.4.0 (OK)
pandas >=0.13.1   :  0.23.4 (OK)
numpy >=1.7       :  1.15.4 (OK)
sympy >=0.7.3     :  1.3 (OK)
cython >=0.21     :  0.29.2 (OK)
qtconsole >=4.2.0 :  4.4.3 (OK)
IPython >=4.0     :  7.2.0 (OK)
matplotlib >=2.0.0:  3.0.2 (OK)
pylint >=0.25     :  2.2.2 (OK)
```

**Comment by ccordoba12** [comment]
This is not a problem with Spyder but with Anaconda. Please report it here

https://github.com/ContinuumIO/anaconda-issues/issues/

**Comment by ccordoba12** [comment]
You can also try

    conda install spyder=3.3.3

but if that doesn't work, please report the problem in Anaconda.

**Comment by jontis** [comment]
Just chiming in here for others that encounter the problem. I use conda in a virtual env, and for some reason, it refused to update spyder from 4.0 to the available 4.1.4
Conda had the 4.1.4 package in the channel, and it was shown in search but did not update through the normal conda update --all or through conda update spyder.
I had to use conda install spyder=4.1.4 to get the update. Probably an error in conda or my use or understanding of it but I still leave the info here to help others.

**Comment by scosoli** [comment]
I am having the same issue here. I have successfully updated from ver.4.0.1 on my imac using  pip install --upgrade spyder, but I am having issues when updating from ver.3.3.6 on my MBP regardless of the way I am doing it (using pip, pip3, conda). it simply refuses to upgrade, despite downloading all the necessary packages and showing up on the list of available versions on anaconda. 
EDIT: in the end it did update the version as @jontis suggested (i.e., conda install spyder=4.1.4), but several packages had to be downgraded and environment inconsistencies fixed before succeeding.
